adhocteam/Head-Start-TTADP
# [TTAHUB-244] Add approved report page to accessibility testing

## Description of change

- add an approved report to the seeders and to the urls we run AXE on
- fix a map that would sometimes not have unique keys
- remove a commented out line of code

## How to test
View the CircleCI output and verify that http://localhost:3000/activity-reports/view/9999 was tested

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-244

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
